docker/docker-compose-db-dev.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  db:
+    container_name: postgres-dev
+    image: postgres:15
+    environment:
+      POSTGRES_PASSWORD: test
+      POSTGRES_USER: postgres
+      POSTGRES_DB: db-dev
+    ports:
+      - 5432:5432

```

Comment:
I had some issues with the port mapping. Since 5432 is a commonly used port maybe we can choose another port to avoid conflict/confusion.